source/includes/wp-api-v3/_product-custom-fields.md

```diff
@@ -2,12 +2,17 @@
 
 The product custom fields API allows you to view the custom field names that have been recorded.
 
-## Product custom fields properties ##
+## Available parameters ##
 
-| Attribute | Type   | Description                                                                                                 |
-| --------- | ------ | ----------------------------------------------------------------------------------------------------------- |
-| `key`     | string | The custom field key.                                                                                       |
-| `value`   | string | The value set for the key.                                                                                  |
+| Parameter | Type     | Description                                                                                                                  |
+| --------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `context`  | string  | Scope under which the request is made; determines fields present in response. Options: `view` and `edit`. Default is `view`. |
+| `page`     | integer | Current page of the collection. Default is `1`.                                                                              |
+| `per_page` | integer | Maximum number of items to be returned in result set. Default is `10`.                                                       |
+| `search`   | string  | Limit results to those matching a string.                                                                                    |
+| `exclude`  | array   | Ensure result set excludes specific IDs.                                                                                     |
```

Comment:
I didn't implement the `exclude` and `include` parameters in the API. Only `context`, `page`, `per_page`, `search` and `order`.

Reply:
Thank you @mdperez86. Done in [4936b6d](https://github.com/woocommerce/woocommerce-rest-api-docs/pull/245/commits/4936b6dd61a16f3ef56c4cbdd782d64ad060d3a8)